bedrock-devnet/devnet/__init__.py

```diff
@@ -160,7 +160,7 @@ def devnet_l2_allocs(paths):
     fqn = 'scripts/L2Genesis.s.sol:L2Genesis'
     # Use foundry pre-funded account #1 for the deployer
     run_command([
-        'forge', 'script', '--chain-id', '901', fqn, "--sig", "runWithAllUpgrades()", "--private-key", "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
+        'forge', 'script', fqn, "--sig", "runWithAllUpgrades()", "--private-key", "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
```

Comment:
How is it that we're able to get away without specifying this?

Reply:
the foundry cheatcode changes the chain ID during execution, and it's loaded from the deploy config.

Reply:
Why do we need to specify the private key here again?

Reply:
See comment below about `msg.sender` so we don't have to specify the private key. It shouldn't be present in any output currently (the account of the `msg.sender` gets dealt with `0` eth before state dump), but I'd rather reduce the scope of possible inputs than try to account for every possible way of running the genesis script.